src/clj/game/cards/icebreakers.clj

```diff
@@ -251,40 +251,48 @@
   "Spends virus counters from any card to pump/break, gains virus counters for successful runs."
   [ice-type]
   {:events {:successful-run {:silent (req true)
-                             :effect (effect (system-msg "adds 1 virus counter to " (:title card))
+                             :effect (effect (system-msg (str "adds 1 virus counter to " (:title card)))
                                              (add-counter card :virus 1))}}
    :abilities [{:label (str  "Break " ice-type " subroutine(s)")
                 :effect (req (wait-for (resolve-ability
                                          state side (pick-virus-counters-to-spend) card nil)
-                                       (do (if-let [msg (:msg async-result)]
-                                             (do (system-msg state :runner
-                                                             (str "spends " msg" to break " (:number async-result)
-                                                                  " " ice-type " subroutine(s)")))))))}
+                                       (when-let* [msg (:msg async-result)
+                                                   n (:number async-result)]
+                                         (system-msg state :runner
+                                                     (str "spends " msg
+                                                          " to break "
+                                                          (quantify n (str ice-type " subroutine")))))))}
                {:label "Match strength of currently encountered ice"
                 :req (req (and current-ice
                                (> (ice-strength state side current-ice)
-                                  (or (:current-strength card) (:strength card)))))
+                                  (or (:current-strength card)
```

Comment:
Feels like this could be made into a nice function like `breaker-strength` or similar.